src/hoc/DataChanges.tsx

```diff
@@ -271,16 +278,18 @@ export const processDataChanges = async (pdc: {
                 AcceptInvitation(remote, upRec.record as InvitationD);
               break;
             case 'organizationmembership':
-              reloadAllOrgs = reloadAllOrgs || await reloadOrgs(upRec.record.id, false);
+              reloadAllOrgs = await reloadOrgs(upRec.record.id, false) || reloadAllOrgs;
               break;
             case 'groupmembership':
-              reloadProjects(upRec.record.id);
+              reloadAllProjects = await reloadProjects(upRec.record.id, false) || reloadAllProjects;
```

Comment:
on line 281 & 284 I am curious why you wouldn't test the Boolean first so you didn't have to call the async function if the Boolean was true?

Reply:
If this logic is required, then you can merge it although I do wonder if you need a comment to explain it.